src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -718,6 +718,8 @@ def aks_create(
     nodepool_allowed_host_ports=None,
     nodepool_asg_ids=None,
     node_public_ip_tags=None,
+    # azure service mesh
+    enable_azure_service_mesh=None,
```

Comment:
If the option is allowed to be enabled when creating (az aks create), what about updating (az aks update)?

Reply:
We have separate `aks mesh enable/disable` command to enable/disable after cluster creation.  Soon after public preview, we are going to add some more sub-commands under `aks mesh`.  Given that, we hope to make these commands all share the same parent command.